.github/pull_request_template.md

```diff
@@ -1,36 +1,42 @@
-### What problem does this PR solve? <!--add and issue link with summary if exists-->
+### What problem does this PR solve?
+<!-- Add an issue link with a summary if exists. -->
 
 ### What is changed and how does it work?
 
-### Check List <!--REMOVE the items that are not applicable-->
+### Check List
+<!-- Remove the items that are not applicable. -->
 
-Tests <!-- At least one of them must be included. -->
+Tests
+<!-- At least one of them must be included. -->
 
- - Unit test
- - E2E test
- - Manual test (add detailed scripts or steps below)
- - No code
+- Unit test
+- E2E test
+- Manual test (add detailed scripts or steps below)
+- No code
 
 Code changes
 
- - Has Go code change
- - Has CI related scripts change
- - Has Terraform scripts change
+- Has Go code change
+- Has CI related scripts change
+- Has Terraform scripts change
```

Comment:
TBH these templates are hardly used and maybe we should remove them...

Reply:
A little verbose I think.